include/network.h

```diff
@@ -43,7 +44,8 @@ enum {
     TLSv1 = 1 << 1,
     TLSv11 = 1 << 2,
     TLSv12 = 1 << 3,
-    SSL_VERSION_MASKS = 1 << 4
+    SSL_VERSION_MASKS = 1 << 4,
+    TLSv13 = 1 << 5
```

Comment:
Would it be possible to remove and disable SSLv3, TLSv1 and TLSv11. Those are all considered unsecure.

Reply:
Sure. I have removed all insecure protocols, so only TLSv1.2 and TLSv1.3 are supported.